src/components/home.js

```diff
@@ -160,6 +160,10 @@ function Home(props) {
             name="title"
             content="Coronavirus Outbreak in India: Latest Map and Case Count"
           />
+          <meta
```

Comment:
Shouldn't this go to index.html?
https://github.com/covid19india/covid19india-react/blob/master/public/index.html#L26